src/nodes/typeGuards/single.ts

```diff
@@ -1,122 +1,245 @@
 import * as ts from "typescript";
 
+export type AnyKeyword = ts.KeywordToken<ts.SyntaxKind.AnyKeyword>;
+export type BigIntKeyword = ts.KeywordToken<ts.SyntaxKind.BigIntKeyword>;
+export type BooleanKeyword = ts.KeywordToken<ts.SyntaxKind.BooleanKeyword>;
+export type FalseKeyword = ts.KeywordToken<ts.SyntaxKind.FalseKeyword>;
+export type ImportKeyword = ts.KeywordToken<ts.SyntaxKind.ImportKeyword>;
+export type NeverKeyword = ts.KeywordToken<ts.SyntaxKind.NeverKeyword>;
+export type NullKeyword = ts.KeywordToken<ts.SyntaxKind.NullKeyword>;
+export type NumberKeyword = ts.KeywordToken<ts.SyntaxKind.NumberKeyword>;
+export type ObjectKeyword = ts.KeywordToken<ts.SyntaxKind.ObjectKeyword>;
+export type StringKeyword = ts.KeywordToken<ts.SyntaxKind.StringKeyword>;
+export type SuperKeyword = ts.KeywordToken<ts.SyntaxKind.SuperKeyword>;
+export type SymbolKeyword = ts.KeywordToken<ts.SyntaxKind.SymbolKeyword>;
+export type ThisKeyword = ts.KeywordToken<ts.SyntaxKind.ThisKeyword>;
+export type TrueKeyword = ts.KeywordToken<ts.SyntaxKind.TrueKeyword>;
+export type UndefinedKeyword = ts.KeywordToken<ts.SyntaxKind.UndefinedKeyword>;
+export type UnknownKeyword = ts.KeywordToken<ts.SyntaxKind.UnknownKeyword>;
+export type VoidKeyword = ts.KeywordToken<ts.SyntaxKind.VoidKeyword>;
```

Comment:
Duplicate changes from #49?

Reply:
Yes. This pr is based of that one as it needs to use som of those functions.